goaway_test.go

```diff
@@ -548,6 +548,36 @@ func TestFalsePositives(t *testing.T) {
 	}
 }
 
+func TestExactWord(t *testing.T) {
+	accept_sentences := []string{
+		"I'm an analyst",
+	}
+	reject_sentences := []string{"Go away, ass."}
+	tests := []struct {
```

Comment:
```suggestion
	accept_sentences := []string{
		"I'm an analyst",
	}
	reject_sentences := []string{"Go away, ass."}
	tests := []struct {
```

Go uses camelCase instead of snake_case - could you make the changes to the other parts of your code?